Simple Interest Calculator

Mo7amedSMahdi opened the pull request:
- Add gitIgnore
- Setup linters
- Add index.html
- Add script.js
- Add Style.css

Mo7amedSMahdi committed:
Add README.md file with full description about the projects

Mo7amedSMahdi committed:
Setup linters for both GitHub Action and Local enviroment

- Add node_modules to gitignore file
- Creat new file:   .github/workflows/linters.yml for GitHub Action Linters
- Create new file:   .gitignore
- Create new file:   .hintrc for WebHint local linter
- Create new file:   .stylelintrc.json for stylelint linter
- new file:   package-lock.json Added by defualt node package install
- new file:   package.json Added by defualt node package install

Mo7amedSMahdi commented on index.html:
```diff
@@ -0,0 +1,59 @@
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+    
```
- [ ] Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.

Mo7amedSMahdi commented on index.html:
```diff
@@ -0,0 +1,59 @@
+                <button onclick="compute()">Compute Interest</button>
+            </div>
+        </div>
+        <br /><br />
```
- [ ] Kindly use padding and margin. This is more semantic to use margin and padding. Chekout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md#:~:text=Use%20the%20appropriate%20tags%20for%20each%20element%20(e.g.%2C%20links%2C%20titles%2C%20etc.)%20and%20use%20HTML5%20semantic%20tags%20(e.g.%2C%20header%2C%20nav)%20over%20divs.)

Mo7amedSMahdi commented on index.html:
```diff
@@ -0,0 +1,59 @@
+
+</body>
+
+</html>
```
Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.

Mo7amedSMahdi replied:
- [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Mo7amedSMahdi reviewed (commented):
# Required Changes ♻️ 
_Check the comments under the review._

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

- [ ] Please fix linter errors ❌ 

![](https://media.giphy.com/media/L422JoNlBBBD42zkQ9/giphy.gif)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BenMukebo reviewed (changes requested):
### Hi **@Mo7amed,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_

-  ***You can use this below as a hint to improve on your second review***

- [ ] Great job on adding linters to your project👏 . However, they are not passing yet... (now add a few words explaining to the student how linters are fair in a project)
- [ ] You most of the time talked about empty spaces, please tag HTML best practices too 😄 
- [ ] Kindly, Add a descriptive README file to your project.... (Now add a few words here too explaining to the student how the readme is important in a project)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mo7amedSMahdi commented on index.html:
```diff
@@ -0,0 +1,59 @@
+                    <option value="7">7</option>
+                    <option value="8">8</option>
+                    <option value="9">9</option>
+                    <option value="10">10</option>
+    
```
- [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Kingstalux replied:
Excellent.

Mo7amedSMahdi commented on index.html:
```diff
@@ -0,0 +1,59 @@
+
+        </span>
+        <br /><br />
+        <footer>
+
+            &#169; Everyone Can Get Rich
+
+
+        </footer>
```
- [ ]  Kindly remove unnecessary extra spaces. Removes Extra spaces make the code looks cleaner.
checkout [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

Mo7amedSMahdi reviewed (commented):
### Hi **@Mo7amed,**

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!

## STATUS: CHANGES REQUESTED  ♻️ 
# Required Changes ♻️ 
_Check the comments under the review._

 [ ] Great job on adding linters to your project👏 . However, they are not passing yet.
- [ ] Kindly, Add a descriptive README file to your project ( Readme is very important to understand your project and how to use it).

![](https://media.giphy.com/media/L422JoNlBBBD42zkQ9/giphy.gif)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Kingstalux reviewed (approved):
Hi @Mo7amedSMahdi , 
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

Good job implementing the required changes 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] (OPTIONAL)  It would be great if you could add emojis to your comments to make them more friendly 😄 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Mo7amedSMahdi commented:
Thanks for mentioning that I will add more emojis to my comment next time.

Mo7amedSMahdi committed:
Change page title

- modified:   index.html Change <title> tag to Intersets insted of interest
- New description line
